README.dev.md

```diff
@@ -0,0 +1,186 @@
+# Developer docs
```

Comment:
I think a short quick start here explaining what is needed day-to-day for key tasks would be useful. I imagine key point is:

- Adding a feature (doing a branch + PR)
- Doing a "release"